src/server/audit/schema.ts

```diff
@@ -51,6 +55,37 @@ export const AuditSchema = z.object({
 
     "update.public-status",
     "update.private-status",
```

Comment:
While you are on this, can you please change these two status with `update.updated`, details will be on summary